### PR TITLE
Bug 1764351 update parameter description

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1764351

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL (tiny change)

File: modules/samples-operator-configuration.adoc [hunk 1]
```diff
@@ -56,7 +56,7 @@ is supported.
 
 |`skippedImagestreams`
 |Imagestreams that are in the Samples Operator’s inventory, but that the cluster
```
Comment (kalexand-rh) [inline modules/samples-operator-configuration.adoc]: no comma

File: modules/samples-operator-configuration.adoc [hunk 2]
```diff
@@ -56,7 +56,7 @@ is supported.
...
-administrator wants the Operator to ignore or not manage.
+administrator wants the Operator to ignore or not manage. You can add a list of imagestream names in this parameter. For example, `["httpd","perl"]`.
```
Comment (kalexand-rh) [inline modules/samples-operator-configuration.adoc]: s/in/to

**Review by kalexand-rh — COMMENTED** [review]
Two picks, then it LGTM

**Comment by bmcelvee** [comment]
No QE review required.

**Pull request merged**